calm/draft/2024-10/meta/core.json

```
@@ -20,6 +20,9 @@
     },
     "controls": {
       "$ref": "control.json#/defs/controls"
+    },
+    "flows": {
+      "$ref": "flow.json#/defs/flows"
     }
```

I wonder here whether we need to connect flows to the architecture or is it enough that flows would be defined separately and would reference the technical architectures they relate to? I think there are arguments both ways.

If we do decide it is valuable to link them, then this would need to be an array.
```suggestion
    "flows": {
      "type": "array",
      "items": {
        "$ref": "flow.json#/defs/flows"
      }
    }
```